components/ProfileMenu.tsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import { ChevronDown } from '@styled-icons/boxicons-regular/ChevronDown';
+import { themeGet } from '@styled-system/theme-get';
+import { get } from 'lodash';
+import { ExternalLink, LogOut, Plus } from 'lucide-react';
+import { FormattedMessage } from 'react-intl';
+import styled, { css } from 'styled-components';
+
+import useLoggedInUser from '../lib/hooks/useLoggedInUser';
+
+import ChangelogTrigger from './changelog/ChangelogTrigger';
+import Avatar from './Avatar';
+import Container from './Container';
+import { Flex } from './Grid';
+import Hide from './Hide';
+import Link from './Link';
+import { withNewsAndUpdates } from './NewsAndUpdatesProvider';
+import PreviewFeaturesModal from './PreviewFeaturesModal';
+import StyledButton from './StyledButton';
+import { Dropdown, DropdownContent } from './StyledDropdown';
+import { Span } from './Text';
+
+const StyledProfileButton = styled(StyledButton)`
+  padding: 0;
+  background-color: white !important;
+`;
+const StyledDropdownContent = styled(DropdownContent)`
+  top: 52px;
+  border-radius: 8px;
+  right: 0;
+  width: 240px;
+
+  padding: 4px 0;
+  hr {
+    margin: 4px 0;
+    border-color: #f3f4f6;
+  }
+  @media screen and (max-width: ${themeGet('breakpoints.0')}) {
+    right: 0;
+    left: 0;
+    width: 100vw;
+    max-width: 100vw;
+    top: 64px;
+    border-radius: 0;
+  }
+`;
+
+const StyledDropdownContainer = styled.div`
+  position: relative;
+  @media screen and (max-width: ${themeGet('breakpoints.0')}) {
+    position: static;
+  }
+`;
+const ProfileHeader = styled.div`
+  padding: 4px 16px;
+  margin-top: 4px;
+  .name {
+    font-size: 14px;
+    color: #0f172a;
+    font-weight: 500;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+  .email {
+    font-size: 12px;
+    color: #64748b;
+    font-weight: 400;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+`;
+
+const StyledMenuEntry = styled(Link)`
+  border: 0;
+  background-color: transparent;
+  color: #334155;
+  padding: 8px 12px;
+  margin: 0 4px;
+  font-size: 14px;
+  border-radius: 4px;
+  white-space: nowrap;
+  vertical-align: middle;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 8px;
+  &:hover {
```

Comment:
We're missing the right cursor on the "Preview Features" and "Log out" options.

![Screenshot from 2023-06-26 10-13-41](https://github.com/opencollective/opencollective-frontend/assets/2119706/ae9715c1-5df5-4083-b8e2-dd97ac4b1b4c)


```suggestion
  cursor: pointer;
  &:hover {
```